src/elements/List/ListList.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getElementType,
+  getUnhandledProps,
+  META,
+  useKeyOnly,
+} from '../../lib'
+
+function ListList(props) {
+  const { children, className } = props
+
+  const ElementType = getElementType(ListList, props)
+  const classes = cx(
+    className,
+    useKeyOnly(ElementType !== 'ul' && ElementType !== 'ol', 'list'),
```

Comment:
Another evil place with class handling, see SUI docs about `ul` &  `ol`.
Combo like `Menu.Menu` is there, too :-\
```jsx
<List>
  <List.Item>
    Item
    <List.List>
      <List.Item>Sub item</List.Item>
    </List.List>
  </List.Item>
</List>
// ...
// <div class="ui list">
//   <div class="item">
//     <div class="list">...</div>
//   </div>
// </div>
// ...
```

```jsx
<List as='ul'>
  <List.Item as='li'>
    Item
    <List.List as='ul'>
      <List.Item as='li'>Sub item</List.Item>
    </List.List>
  </List.Item>
</List>
// ...
// <ul class="ui list">
//   <li>
//     <ul>...</ul>
//   </li>
// </ul>
// ...
```